Dockerfile

```diff
@@ -4,7 +4,10 @@
 # SPDX-License-Identifier: Apache-2.0
 
 # Prepare to cross-compile by always running the build stage in the build platform, not the target platform.
```

Comment:
This comment belongs with the FROM statement.

Reply:
Ah yeah thats true. I'll move the ARGs up.